src/Test.UnitTests.Sarif/TestData/SarifCurrentToVersionOneVisitor/ExpectedOutputs/OneRunWithInvocationAndNotifications.sarif

```diff
@@ -27,7 +27,6 @@
           "message": "Exception evaluating rule \"C2152\". Rule disabled; run continues.",
           "level": "error",
           "threadId": 52,
-          "time": "2016-07-16T14:18:43.119Z",
```

Comment:
>time [](start = 11, length = 4)

this should still exit in v2 as timeUtc maybe?

Reply:
it's missing in the transformer!!

---
In reply to: [287543306](https://github.com/microsoft/sarif-sdk/pull/1501#discussion_r287543306) [](ancestors = 287543306)

Reply:
thanks, i will take it up as a bug, logged: [https://github.com/microsoft/sarif-sdk/issues/1503](https://github.com/microsoft/sarif-sdk/issues/1503)

---
In reply to: [287543510](https://github.com/microsoft/sarif-sdk/pull/1501#discussion_r287543510) [](ancestors = 287543510,287543306)